lib/tables.dart

```diff
@@ -127,8 +144,39 @@ class Table<T> {
     // Re-build the table.
     final List<Element> rowElements = <Element>[];
 
-    for (T row in rows) {
+    int firstRenderedRowInc = 0;
```

Comment:
It makes the names longer, but we should avoid abbreviations (`Inclusive`, `Exclusive`)